src/library/process.js

```diff
@@ -35,8 +35,19 @@ export const processMessage = async (AWS, region, service, account,
         ':serv': { S: `${service}` },
       },
     },
-  );
+    );
+    
+  // CR: Mickey: we should think about whether it's preferable to use graphQl here
```

Comment:
@MickeyMcc I want to discuss and change the whole context object in the later iterations. If the context object size keeps increasing due to whatever reasons like addition of more products and such, it will one day cross the 256KB limit enforced on the SNS and SQS messages. We need to be able to expose methods to the worker to get the context data (at least as par as the user, account, product and tile objects are concerned) through GraphQL/ direct DynamoDB queries as part of the framework.